src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -313,6 +314,7 @@ public void beforeCommand(CommandEnvironment env) throws AbruptExitException {
     }
 
     this.remoteOptions = remoteOptions;
+    this.env = env;
```

Comment:
nit: add `checkState(this.env == null)` at the beginning.

Reply:
Done!